src/app/app.component.scss

```diff
@@ -1,4 +0,0 @@
-img{
-  width: 100%;
```

Comment:
css not required?

Reply:
yes not required.. removed background image